server/src/main/java/com/coincoinche/engine/cards/Card.java

```diff
@@ -15,6 +17,7 @@ public String getShortName() {
   }
 
   @Override
+  @JsonValue
```

Comment:
This tells Spring that this toString method can be used to return the JSON representation of a card (i.e. 2 letters).

Reply:
Very nice! Could we use `getShortName` for this? They are the same for now but `toString` is more for us to have a human-readable string representation for cards, right?